ovh/types_iploadbalancing.go

```diff
@@ -669,3 +674,72 @@ func (opts *IpLoadbalancingVrackNetworkUpdateOpts) FromResource(d *schema.Resour
 
 	return opts
 }
+
+type IpTaskStatus string
+
+const (
+	IpTaskStatusCancelled     IpTaskStatus = "cancelled"
+	IpTaskStatusCustomerError IpTaskStatus = "customerError"
+	IpTaskStatusDoing         IpTaskStatus = "doing"
+	IpTaskStatusDone          IpTaskStatus = "done"
+	IpTaskStatusInit          IpTaskStatus = "init"
+	IpTaskStatusOvhError      IpTaskStatus = "ovhError"
+	IpTaskStatusTodo          IpTaskStatus = "todo"
+)
+
+type IpTask struct {
+	Comment     string       `json:"comment"`
+	Destination string       `json:"destination"`
```

Comment:
from what I see in the API schemas here: https://api.ovh.com/console-preview/?section=%2Fip&branch=v1#post-/ip/-ip-/move
the field `destination` is an object containing a `serviceName` field